addon/index.js

```diff
@@ -26,8 +30,8 @@ var counter = 0;
   @extends DS.Adapter
 */
 export default DS.Adapter.extend({
-  // by default, fixtures are already in normalized form
-  serializer: null,
+  defaultSerializer: '-default',
```

Comment:
:dancers: I love this change. Although I wonder if it will break places where the Fixture adapter is being used in tests and for an app that has a custom `serializer:application`.